examples/trials/mnist-nas/classic_mode/config_hopt.yml

```diff
@@ -0,0 +1,16 @@
+authorName: default
+experimentName: example_mnist
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote, pai
+trainingServicePlatform: local
+#choice: true, false
+useAnnotation: true
+tuner:
+  builtinTunerName: TPE
+trial:
+  command: python3 mnist.py --batch_num 200
+  codeDir: .
+  gpuNum: 0
+  nasMode: classic_mode
```

Comment:
could you refine the doc about "classic_mode" and ... because it hard to understand...

Reply:
will have refactor for NAS support including document in the following week, let's do doc improvement when doing refactor.